ConfigurationSystem/Agent/Bdii2CSAgent.py

```diff
@@ -304,11 +305,16 @@ def __purgeSites(self, ceBdiiDict):
       if not ces:
         self.log.error("No CE information for site:", site)
         continue
-      res = getCESiteMapping(ces[0])
-      if not res['OK']:
-        self.log.error("Failed to get DIRAC site name for ce", "%s: %s" % (ces[0], res['Message']))
-        continue
-      siteInCS = res['Value'][ces[0]]
+      siteInCS = 'Not_In_CS'
+      for ce in ces:
+        res = getCESiteMapping(ce)
```

Comment:
Calling `getCESiteMapping()` (without input `ce`) would get you the whole mapping. You can call it outside of the loop.